ural/should_be_rss_url.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Ural Should Be RSS URL
+# =============================================================================
+#
+# Function returning whether the given url should be a rss feed url.
+#
+import re
+
+from ural.utils import urlpathsplit
+from ural.utils import urlsplit
+import ural.get_hostname
+
+PATTERN_QUERY = re.compile(r"(?:format|type)\=(?:xml|feed|atom|rss)", re.I)
```

Comment:
In `ural` compiled regex constants end with `_RE` and `PATTERN` (as a suffix, not a prefix) is reserved for uncompiled pieces of regexes.